textile/features.textile

```diff
@@ -2152,6 +2168,13 @@ class Message: // TM*
   id: String // TM2a
   name: String? // TM2g
   timestamp: Time // TM2f
+  action: enum // TM2j
+  serial: string // RTN7b, TM2K
```

Comment:
The serial here is different to `RTN7b` afaik - `RTN7b` is a `ProtocolMessage` field. In fact, the `msgSerial` of the `ProtocolMessage` is a uint, not a timeserial.

Reply:
Ditto on the line below

Reply:
You're quite correct, I've updated this to point to the `@serial@` field I've added to the `@Message@` definition.